_pages/about-us/teams/acqstack.md

```diff
@@ -1,48 +1,118 @@
+
+### Our Principles
+
+* Use **human-centered design** to focus on software that meets actual people's needs/
+* Leverage **open-source software**, and select modern tech stacks to ensure long-term sustainability.
+* Utilize **product thinking** to prioritize, focus work, and deliver demonstrable value.
+* Structure contracts to **support modularity, lower costs, increase quality, and avoid vendor lock-in.**
+* Ensure that **agile development methodologies** are employed to deliver working code sooner.
+* **Communicate clearly** about methods, lessons-learned, and recommendations.
+
+### What We Do
+
+We focus on procurements of intentionally modest scope for digital products and services (procures and deliver incrementally), so that we can deliver end-user value more rapidly, assess the results more quickly, and refine our approaches over time.
 
 In pursuit of this strategy, we:
 
-- **Buy digital products and services for our own organization (TTS) and for partner agencies** to pilot our own procurement tools and processes, and to demonstrate that compliance can still be achieved in the context of more efficient and effective acquisition approaches;
-- **Develop and pilot novel methods of working with industry** which engage vendors of all sizes and encourages them to compete in the market and deliver better results; and
-- **Consult with clients on acquisition strategy, and provide post-award coaching and support** rooted in real project work to assist our clients in the development of the skills needed to be more savvy consumers of digital products and services moving forward, and ensure their long-term success beyond the scope of our work together.
+* Consult with our agency partners to help them buy digital products and services  demonstrating that compliance can still be achieved in the context of more efficient and effective acquisition approaches.
+* Develop innovative acquisition strategies within the confines of the Federal Acquisition Regulations (FAR). 
+* Develop and pilot novel methods of working with industry and encourage vendors of all sizes to compete for work and offer their unique skills to government. 
+* Work in the open. We strive to publicly share as much information about our work with agencies partners as possible. Vendors we partner with commit to using open-source technologies.
+* Consult with partner agencies on acquisition strategy, conduct workshops, provide pre-award and post-award coaching. Our work is rooted in real project work to assist our partner agencies to develop the skills needed to be more savvy consumers of digital products and services moving forward, and ensure their long-term success beyond the scope of our work together.
+
+### How we communicate
+
+The majority of our conversations take place on Slack and video.  There are a number of slack channels for acquisition discussions. 
```

Comment:
If were going to say there are a bunch of Slack channels, maybe we should mention some of the more popular ones? And if we're not mentioning any, I'm not sure the sentence adds value. If we keep it though, Slack should be capitalized.